### DISCUSSION TITLE
Adding tools to Mise manager

### DISCUSSION THREAD

**Discussion by mtweeman** [renovatebot/renovate#32328] [Suggest an Idea]
Hey,

currently, I'm using Mise to manage packages in my project. For now, a bunch of them is supported by Renovate. Documentation clearly states that for the new ones [PR is needed](https://docs.renovatebot.com/modules/manager/mise/#plugintool-support). I'd like to extend this support for new tools/plugins (already existing in asdf/Mise):
k3s
oci
atmos
packer
I forked the repo and prepared what's needed in `lib/modules/manager/asdf/upgradeable-tooling.ts`. What else should I change before raising PR?

**Comment by rarkins** [comment]
That's about all. Ideally run the linting tests first before pushing, to save yourself time waiting for CI/CD to be approved for first-time contributors. Also a good reference can be to look at past contributions to asdf, which are similar: https://github.com/renovatebot/renovate/commits/main/lib/modules/manager/asdf

**Reply by mtweeman** [reply]
I did as you suggested. Here's the [PR](https://github.com/renovatebot/renovate/pull/32333). Should I link this discussion to it as well?
